packages/runner-ct/src/header/header.tsx

```diff
@@ -9,11 +9,10 @@ import State from '../lib/state'
 import { configFileFormatted } from '../lib/config-file-formatted'
 import SelectorPlayground from '../selector-playground/selector-playground'
 import selectorPlaygroundModel from '../selector-playground/selector-playground-model'
-import { ExtendedConfigOptions } from '../app/RunnerCt'
 
 interface HeaderProps {
   state: State
-  config: ExtendedConfigOptions
```

Comment:
This type doesn't seem to exist anywhere. We should try to enable `strict` asap.